tests/base/test_logging.py

```diff
@@ -1,34 +1,34 @@
 import pytest
 import pydp as dp
 
+# TODO
+# class TestLogging:
+#     def test_logging(self):
+#         directory = "tests"
+#         vlog_level = 3
+#         success = dp.Logging(directory, "log_test", vlog_level)
+#         assert success is True
```

Comment:
Can we mention in TODO that a read_only_property needs to be added in `logging.cc` file?

Reply:
Sure